### PR TITLE
fix indexes and urls for all (rogue and bard) expertise options

### PR DESCRIPTION
**Pull request by Freedzio**
## What does this do?
Fixes option in expertise giving features

## How was it tested?
\

## Is there a Github issue this is resolving?
#265 

## Did you update the docs in the API? Please link an associated PR if applicable.
\

## Here's a fun image for your troubles
random photo - update me

### CODE REVIEW

File: src/5e-SRD-Features.json [hunk 1]
```diff
@@ -519,96 +519,96 @@
     "choice": {
       "choose": 2,
       "type": "feature",
-      "from": [
+      "from":  [
```
Comment (fergcb) [inline src/5e-SRD-Features.json]: Dodgy spacing here, same below. Otherwise, LGTM!

```suggestion
      "from": [
```

Comment (Freedzio) [reply src/5e-SRD-Features.json]: fixed :)

File: src/5e-SRD-Features.json [hunk 2]
```diff
@@ -521,94 +521,94 @@
       "type": "feature",
       "from": [
         {
-          "index": "bard-expertise-survival",
```
Comment (bagelbits) [inline src/5e-SRD-Features.json:524]: Uh..... these are all Bard expertise?

File: src/5e-SRD-Features.json [hunk 3]
```diff
@@ -1139,94 +1139,94 @@
...
```
Comment (bagelbits) [inline src/5e-SRD-Features.json:1142]: These are all bard expertise too.

Comment (Freedzio) [reply src/5e-SRD-Features.json:1142]: oopsie. fix is on the way

**Review by bagelbits — APPROVED** [review]
LGTM!

**Pull request merged**